Makefile

```diff
@@ -11,8 +19,15 @@ build-assets:
 start-worker:
 	go run worker/main.go
 
+db-migrate:
+	bee migrate -driver=postgres -conn="$(DATABASE_URL)"
+
+db-rollback:
+	bee migrate rollback -driver=postgres -conn="$(DATABASE_URL)"
+
 dev:
 	docker-compose -f docker-compose.dev.yml up -d
+	make db-migrate
```

Comment:
@junan Won't you have any issues with `database not found` or something about database issues around here?

cc: @carryall

Reply:
@Lahphim @junan The issue I got earlier is when we build the database container for the first time and run the migration right after the `docker-compose` command the migration command would fail because it cannot find the database yet. So I have to provide 2 make commands, one for preparing the database container and another one for the migration. 💭

Reply:
![Screen Shot 2021-04-09 at 3 22 58 PM](https://user-images.githubusercontent.com/6076762/114151620-7df70300-9947-11eb-965c-0fa9d13390fb.png)

@Lahphim Its works fine for me locally and staging